README.md

```diff
@@ -177,6 +177,40 @@ Glob patterns are expanded according to the following rules:
    separators)
  - `?`, `*` and `**` do not match a `.` at the beginning of a file/directory
 
+### Not repeating yourself
+
+It is possible to use YAML [anchors][yaml-anchor] (`&`), [aliases][yaml-alias]
+(`*`) and [merge keys][yaml-merge] (`<<`) to define fields and reference them
+later.
+
+```yaml
+executables:
+  my-exe-1: &my-exe
+    main: my-exe-1.hs
+    dependencies: [base, my-lib]
+    ghc-options: [-threaded]
+  my-exe-2:
+    <<: *my-exe
+    main: my-exe-2.hs
+```
+
+Warnings for unknown fields will not be emitted for top-level fields starting
+with an underscore, so you can declare global aliases too:
+
+```yaml
+_: &exe-ghc-options
```

Comment:
Let's use a variation of the common convention where the field name and anchor name are the same:

```yaml
_exe-ghc-options: &exe-ghc-options
  - ...
```